hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/service/MonitorService.java

```diff
@@ -410,11 +411,12 @@ public void transition(final ServiceController<? extends Object> controller,
             Object jndiObject = getHawkularMonitorContext();
             ContextNames.BindInfo bindInfo = ContextNames.bindInfoFor(jndiName);
             BinderService binderService = new BinderService(bindInfo.getBindName());
-            Injector<ManagedReferenceFactory> managedObjectInjector =
-                    WildflyCompatibilityUtils.getManagedObjectInjectorFromBinderService(binderService);
-            Injector<ServiceBasedNamingStore> namingStoreInjector =
-                    WildflyCompatibilityUtils.getNamingStoreInjectorFromBinderService(binderService);
-            ManagedReferenceFactory valueMRF = WildflyCompatibilityUtils.getImmediateManagedReferenceFactory(jndiObject);
+            Injector<ManagedReferenceFactory> managedObjectInjector = WildflyCompatibilityUtils
+                    .getManagedObjectInjectorFromBinderService(binderService);
+            Injector<ServiceBasedNamingStore> namingStoreInjector = WildflyCompatibilityUtils
+                    .getNamingStoreInjectorFromBinderService(binderService);
+            ManagedReferenceFactory valueMRF = WildflyCompatibilityUtils
+                    .getImmediateManagedReferenceFactory(jndiObject);
```

Comment:
these lines I didn't touch. But the code formatter is reformatting them. I don't know why our formatters are different. But this trivial is just formatting, no structural change.